src/models/UserMapper.php

```diff
@@ -642,4 +642,15 @@ public function resetPassword($token, $password)
 
         return false;
     }
+
+    public function delete($user_id)
+    {
+        try {
+            $sql = "delete from user where ID = :user_id";
+            $stmt = $this->_db->prepare($sql);
+            return $stmt->execute(array("user_id" => $user_id));
```

Comment:
Sorry for the slow reply.

I think it should be done now as the user id should be purged.